sympy/solvers/tests/test_solveset.py

```diff
@@ -1724,6 +1731,130 @@ def test_solve_nonlinear_trans():
     assert nonlinsolve([x**2 - y**2/exp(x)], [x, y]) == soln4
 
 
+def test_issue_14642():
+    x = symbols('x',dtype = complex)
```

Comment:
What does `dtype=complex` do?

Reply:
No idea. The original code was like this...

Reply:
I guess it is supposed to be `complex=True` or something. It should be removed though because it doesn't do anything (ideally this would raise an error).

Reply:
OK! Will do!